experiments/list_unsupported.sh

```diff
@@ -40,11 +40,27 @@ done
 
 echo "$1: Unsupported features\n" > "$file_name".txt
 
+# gnat on the path?
 if ! command -v gnat > /dev/null; then
    echo >&2 "Gnat not on PATH!"
+   # Take a wild guess at where gnat might be installed...
```

Comment:
instead of doing this, how about checking if `locate` is on the path and using that to (maybe) find gnat if it is?

Reply:
Good idea but I worry that it would find the system GNAT rather than the one we want (the 2016 one).  I know Trevor and I both have multiple versions of GNAT installed.

Reply:
I didn't mean to suggest that this should be used automatically as the path, just as a guide to give a more targeted suggestion along the lines of "hey, did you mean to add <this path here> to your PATH?"

Reply:
`locate` is also not posix standard I think (so it might not be on the target system, or if it is, might not be enabled - e.g. OSX has the locate command, but doesn't run the indexer). My main purpose here is to help someone who is following the documented install process, which recommends this location for gnat.